### PR TITLE
support dark mode

### LINKED ISSUE #1: dark mode support
Hi @polemius , would be cool if this tool had dark mode-friendly colours. Currently a struggle to read console output unless I switch to light mode.

Mind if I open a PR to support dark?

Cheers!

### PR DESCRIPTION
**Pull request by jezzzm**
resolves #1 
i tried not to go too overboard with the colours!

dark mode:
image

light mode:
image

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,28 +1,59 @@
...
     console.log(
-      '%cPlease add: %c`persistence_UNSTABLE: { type: "log" }` %cto atom object to see the values',
-      'color: #333',
-      'color: #000',
+      '%cPlease add: %c`persistence_UNSTABLE: { type: "log" }` %cto see the values for atom object: ',
+      colors.base,
+      colors.info,
+      colors.base,
       action.name,
-      'color: #333',
```
Comment (jezzzm) [inline index.js:19]: this was printing the string `color: #333` to console so I moved it above `action.name`

File: index.js [hunk 2]
```diff
@@ -1,28 +1,59 @@
...
     action.name,
   )
   if (action.persistence) {
-    console.log('%cValue of atom cannot be read', 'color: #f00')
+    console.log('%cValue of atom cannot be read', colors.error)
     console.log(
-      '%cPlease add: %c`persistence_UNSTABLE: { type: "log" }` %cto atom object to see the values',
-      'color: #333',
-      'color: #000',
+      '%cPlease add: %c`persistence_UNSTABLE: { type: "log" }` %cto see the values for atom object: ',
```
Comment (jezzzm) [inline index.js:46]: small change to wording so you know the following value, `action.name` refers to an atom obj

**Comment by jezzzm** [comment]
just realised there's a utils.js file, these change could probably go there instead

**Comment by polemius** [comment]
@jezzzm Looks very great! :+1: Thank you for PR. Now I am out of my computer so I will merge and release later.

**Comment by polemius** [comment]
Released in 0.4.0 🎉 🚀 @jezzzm Thanks you for contribution! Do you have a twitter account? I will mention you in tweet about new release

**Comment by jezzzm** [comment]
nice! twitter handle is @jezmilledge. my tweet about this was just retweeted by @vjeux so i think this repo will soon have lots more traffic 👍

File: index.js [hunk 3]
```diff
@@ -1,28 +1,59 @@
...
+    colors.previous += '#444'
+    colors.base += '#000'
+  }
+  return colors
+}
+
+const setColors = () => {
+  const isDark =
+    window.matchMedia &&
+    window.matchMedia('(prefers-color-scheme: dark)').matches
```
[the remark this answers is not in the document]
Comment (jezzzm) [reply index.js:28]: this fn is called prior to each logging instance so it still uses correct colours for any logs immediately following a change.

I did consider adding a listener but thought it was simpler just to call it each time. happy to update it if you reckon it's a better implementation.

**Comment by polemius** [comment]
@jezzzm awesome! Thanks! 👍

**Comment by vjeux** [comment]
I hope this is fine, it’s a really cool project :)

**Pull request merged**